src/core/deltafilewrapper.cpp

```diff
@@ -996,9 +996,18 @@ void DeltaFileWrapper::setIsPushing( bool isPushing )
 QJsonValue DeltaFileWrapper::geometryToJsonValue( const QgsGeometry &geom ) const
```

Comment:
Interesting we have these changes in this PR?

Reply:
Weird, fixed.